src/data/__tests__/calculate.test.ts

```diff
@@ -1,12 +1,17 @@
 import {
   CalculatorData,
+  DAY_0,
   calculate,
   calculateLocationPersonAverage,
   defaultValues,
 } from 'data/calculate'
 import { BUDGET_ONE_PERCENT, RiskProfile, RiskProfileEnum } from 'data/data'
 import { prepopulated } from 'data/prepopulated'
 
+const MS_PER_DAY = 1000 * 60 * 60 * 24
+const BASE_RATE = 0.003
+const RATE = 0.006
```

Comment:
I don't understand the meaning of these or why they were chosen. Could you explain their purpose with a comment?

Reply:
Done